### PR TITLE
Added unit tests

### PR DESCRIPTION
**Pull request by ACscooter**
Additional changes include
- Clarified comments and documentation
- Updated client API parameters to match latest mixer release

### CODE REVIEW

File: datacommons/core.py [hunk 1]
```diff
@@ -95,33 +93,36 @@ def get_property_values(dcids,
...
-  The return value is a list of tuples (s, p, o) where s denotes the subject
-  entity, p the property, and o the object.
+  The return value is a dictionary mapping given dcids to list of triples. The
+  triples are repsented as 3-tuples (s, p, o) where "s" denotes the subject, "p"
+  the property, and "o" the object.
 
   Args:
-    dcid: A list of dcids to get triples for.
+    dcids: A list of dcids to get triples for.
     limit: The maximum number of triples to get for each combination of property
```
Comment (tjann) [inline datacommons/core.py]: can say property label to be more clear

File: datacommons/__init__.py [hunk 2]
```diff
@@ -13,13 +13,13 @@
 # limitations under the License.
 """Data Commons module."""
 
-from __future__ import absolute_import
-from __future__ import division
```
Comment (tjann) [inline datacommons/__init__.py]: no matter how I refresh, I still see this as deleted. will come back to it but can you check that this was put back?

Comment (ACscooter) [reply datacommons/__init__.py]: Added back

File: datacommons/core.py (hunk 1 above)
Comment (ACscooter) [reply datacommons/core.py]: Will clarify with a section on what a triple is.

**Pull request merged**